src/ripple/protocol/LedgerFormats.h

```diff
@@ -249,9 +249,8 @@ enum LedgerSpecificFlags {
         0x10000000,               // True, reject new paychans
     lsfDisallowIncomingTrustline =
         0x20000000,               // True, reject new trustlines (only if no issued assets)
-    lsfAMM = 0x40000000,          // True, AMM account
     lsfAllowTrustLineClawback =
-        0x80000000,               // True, enable clawback
+        0x40000000,               // True, enable clawback
```

Comment:
We need to make sure this change is well publicized.

Reply:
What is the process for this? This feature is pending 12.1.0 release.

Reply:
Is there any way we can keep the clawback flag the same as it was before?

Client libraries have already merged support for Clawback, [implementing the prior value](https://github.com/XRPLF/xrpl.js/pull/2353/files#diff-ecaed36481e2b2545d4bc7180df7092251f25835586b90d837be4dd11dceed62R208).

Reply:
Sure, I'll change it back.

Reply:
The ideal is to change it back.

For future reference: if the flag must be changed, the process is to put the `API Change` label on the PR. (Come to to think of it - there are other API Changes in this PR, so I'll put that label on regardless. They only affect APIs which have not been released yet, but I know people have already been implementing/testing AMM, so we should publicize the changes for their sake.) While it's the responsibility of client library maintainers and API integrators to review release notes and look at the API Changelog, in cases where we know what the relevant/impacted apps/projects are, it would be nice courtesy to inform them directly as well.

Reply:
Woops. Thought I'd posted this earlier:

The big issue is that the flag is specified in the Clawback spec (https://github.com/XRPLF/XRPL-Standards/tree/master/XLS-39d-clawback#32-account-root-modifications). Plus, I'm sure there are downstream projects that have already started implementing Clawback support using this value.

I think it makes the most sense to change this back and leave a comment explicitly saying that `0x40000000` is available.